tasties/urls.py

```diff
@@ -13,10 +13,9 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.contrib import admin
 from django.urls import path
 from tasties_app import views
-
+from django.contrib import admin
```

Comment:
You remove this line and then adding it again

Reply:
I remove the change from the urls file